velox/functions/sparksql/RegexFunctions.h

```diff
@@ -60,4 +60,18 @@ std::shared_ptr<exec::VectorFunction> makeRegexExtract(
     const std::vector<exec::VectorFunctionArg>& inputArgs,
     const core::QueryConfig& config);
 
+/// Full implementation of RegexReplace found in SparkSQL only,
+/// due to semantic mismatches betweeen Spark and Presto
```

Comment:
typos or incomplete sentence; also this comment seems to be duplicated; let's not duplicate comments like this to avoid diverging

Reply:
Removed